grid/address/geocoding/maps/tomtom.suma

```diff
@@ -0,0 +1,136 @@
+profile = "address/geocoding@3.1"
+provider = "tomtom"
+
+map Geocode {
+  payload = input.query || `${input.streetAddress || ""} ${input.addressLocality || ""} ${input.addressRegion || ""} ${input.postalCode || ""} ${input.addressCountry || ""}`;
+
+  http GET "/search/2/geocode/{payload}.json" {
+    security "apikey"
+
+    request {
+      query {
+        storeResult = false
```

Comment:
`storeResult` query parameter is optional, default value is `false` and it is deprecated. According to the doc since 2018 the response status code for queries with `storeResult` param should be 400 Bad request.

Reply:
fixed, thanks!